sessionctx/variable/sysvar_test.go

```diff
@@ -318,3 +322,227 @@ func (*testSysVarSuite) TestCollationServer(c *C) {
 	c.Assert(sv.SetSessionFromHook(vars, "utf8mb4_bin"), IsNil)
 	c.Assert(vars.systems[CharacterSetServer], Equals, "utf8mb4") // check it also changes charset.
 }
+
+func (*testSysVarSuite) TestTimeZone(c *C) {
+	sv := GetSysVar(TimeZone)
+	vars := NewSessionVars()
+
+	// TiDB uses the Golang TZ library, so TZs are case-sensitive.
+	// Unfortunately this is not strictly MySQL compatible. i.e.
+	// This should not fail:
```

Comment:
I think we could add `TODO: ` marker, which is much easier to search, despite if there is an issue.

Reply:
I've added a link to the issue directly. https://github.com/pingcap/tidb/issues/8087